add-domain

```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
+mail-e9eredelivery.com
+trackpp.searchmyitem.xyz
+steamcomnnulty.com
+yahoofbuyv.weebly.com
```

Comment:
Taken down, See also Special Rules in Pyfunceble 

https://github.com/funilrys/PyFunceble/blob/89f2b69c4f02657124d3a05d6b73e43ca5a46c06/PyFunceble/checker/availability/extra_rules.py#L99-L101

from commit https://github.com/funilrys/PyFunceble/commit/89f2b69c4f02657124d3a05d6b73e43ca5a46c06